src/main/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -139,12 +161,12 @@ public TribbleIndexedFeatureReader(final String featureFile, final FeatureCodec<
     private void loadIndex() throws IOException{
         String indexFile = Tribble.indexFile(this.path);
         if (ParsingUtils.resourceExists(indexFile)) {
```

Comment:
Will `ParsingUtils.resourceExists()` return true for an index that lives on GCS?